### ISSUE TITLE
site menu should be within `<nav>` tag

### ISSUE THREAD

**Issue by Lane** [Hyperobjekt/untd-site#109]
currently only the menu button is within the nav tag, but all menu links should be within the `<nav>` element.

![image](https://user-images.githubusercontent.com/21034/122300119-bc62ee00-ceb3-11eb-9b30-549288188220.png)

![image](https://user-images.githubusercontent.com/21034/122299688-2b8c1280-ceb3-11eb-879b-11847a0911ce.png)

see [All page content must be contained by landmarks](https://dequeuniversity.com/rules/axe/4.2/region?application=AxeChrome)
